deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -662,13 +662,16 @@ def merge_input_provider(index, item):
     )
 
     logger.info(
-        f"partition-{params.source_partition_locator.partition_values},"
+        f"Partition-{params.source_partition_locator.partition_values},"
         f"compacted at: {params.last_stream_position_to_compact},"
     )
+    logger.info(
+        f"Checking if partition {rcf_source_partition_locator} is inplace compacted against {params.destination_partition_locator}..."
+    )
     is_inplace_compacted: bool = (
-        params.source_partition_locator.partition_values
+        rcf_source_partition_locator.partition_values
```

Comment:
I don't see how the existing rebase then incremental test case is passing. Please investigate.

Reply:
I believe it's because the rebase-then-incremental test cases do not set `source_partition_locator` and `destination_partition_locator` to the same value, hence no chance of being marked as in-place compacted.

Reply:
Got it. A separate rebase then incremental where source and destination are same partition locator makes sense?